doc/man1/flux-config.rst

```diff
@@ -90,7 +90,7 @@ reload
 .. program:: flux config reload
 
 :program:`flux config reload` tells :man1:`flux-broker` to reload its TOML
-configuration.
+configuration after they have been modified.
```

Comment:
either "after it has been modified" or "... configuration files after they have been reloaded"?